packages/core/src/carousel/private/utils/useInterval.ts

```diff
@@ -11,11 +16,24 @@ const useInterval = (callback: () => void, delay: number) => {
     function tick() {
```

Comment:
What if memorize this function or push outside of the component? Might improve performance.